cri/v1alpha1/cri.go

```diff
@@ -1000,9 +1000,11 @@ func (c *CriManager) ImageStatus(ctx context.Context, r *runtime.ImageStatusRequ
 
 	imageInfo, err := c.ImageMgr.GetImage(ctx, ref.String())
 	if err != nil {
-		// TODO: separate ErrImageNotFound with others.
-		// Now we just return empty if the error occurred.
-		return &runtime.ImageStatusResponse{}, nil
+		if errtypes.IsNotfound(err) {
```

Comment:
why return nil error here? could we check the IsNotfound out of this scope?

Reply:
The CRI API declares this :
```
ImageStatus returns the status of the image. If the image is not present, returns a response with ImageStatusResponse.Image set to nil.
```